### PR TITLE
Add How to document

### PR DESCRIPTION
**Pull request by lajanda**
Add How to document. Note that the document has been uploaded in an issue.

### CODE REVIEW

File: content/resources.md [hunk 1]
```diff
@@ -19,3 +19,7 @@ template = "page.html"
   - Wylie, Christopher. 2019. *Mindf'ck: Cambridge Analytica and the plot to break America*. New York: Random House.
   - Jankowicz, Nina. 2020. *How to Lose the Information War: Russia, Fake News, and the Future of Conflict.*
     I. B. Tauris & Company, Limited.
+
+### How to documents for the THREAT-DEFUSER team
+
+- [How to register publications and other activities in CRISTIN](How to register publications and other activities.docx)
```
Comment (bast) [inline content/resources.md:25]: But where does the link point to? Where is that docx file?

Comment (bast) [reply content/resources.md:25]: Aha file is part of #22.

**Comment by lajanda** [comment]
Sorry, I was dumb, I tried to upload the file first and then do the changes. If I had tried to make the changes first, I would have seen that I could put the file in the pull request. I will try to do better in the future!
            --laura

**Comment by bast** [comment]
No problem. I will merge this now and send a follow-up fix and ping you in there.

**Pull request merged**